quarkdown-html/src/main/scss/global.scss

```
@@ -216,3 +216,15 @@
   --r-block-margin: var(--qd-block-margin);
   --r-heading-margin: var(--qd-heading-margin);
 }
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)